diff --git a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
index 540078a..1db14fd 100644
--- a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
+++ b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
@@ -546,9 +546,15 @@ exports.LoadUtils = () => {
                     messageSecret: window.crypto.getRandomValues(
                         new Uint8Array(32),
                     ),
-                    cannotBeRanked: window
-                        .require('WAWebStatusGatingUtils')
-                        .canCheckStatusRankingPosterGating(),
+                    cannotBeRanked: (() => {
+                        try {
+                            return window
+                                .require('WAWebStatusGatingUtils')
+                                .canCheckStatusRankingPosterGating();
+                        } catch (e) {
+                            return false;
+                        }
+                    })(),
                 },
             );
 
